spec/rails_helper.rb

```diff
@@ -5,7 +5,10 @@
 # Prevent database truncation if the environment is production
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
+
 # Add additional requires below this line. Rails is not loaded until this point!
+require 'view_component/test_helpers'
```

Comment:
The changes in this file are already present in `spec/support/components.rb`

Reply:
Indeed :) missed that. Updated